arch/arm/core/aarch32/cortex_a_r/exc.S

```diff
@@ -38,30 +38,12 @@ GTEXT(z_arm_undef_instruction)
 GTEXT(z_arm_prefetch_abort)
 GTEXT(z_arm_data_abort)
 
-/**
- * @brief Undefined instruction exception handler
- *
- * An undefined instruction (UNDEF) exception is generated when an undefined
- * instruction, or a VFP instruction when the VFP is not enabled, is
- * encountered.
- */
-SECTION_SUBSEC_FUNC(TEXT, __exc, z_arm_undef_instruction)
-	/*
-	 * The undefined instruction address is offset by 2 if the previous
-	 * mode is Thumb; otherwise, it is offset by 4.
-	 */
-	push {r0}
-	mrs r0, spsr
-	tst r0, #T_BIT
-	subeq lr, #4	/* ARM   (!T_BIT) */
-	subne lr, #2	/* Thumb (T_BIT) */
-	pop {r0}
-
+.macro exception_entry, mode
```

Comment:
IIRC, GNU assembler macro syntax is supposed to be something like `.macro [name] [param1], [param2], ...`, so there shouldn't be a comma after the macro name.

Reply:
fixed